packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -61,21 +77,37 @@ class ScaffoldGeometry {
   /// This is null when there is no floating action button showing.
   final Rect floatingActionButtonArea;
 
+  /// A [NotchMaker] for the floating action button.
```

Comment:
maybe point to [NotchMaker] for the documentation of what the contract is

Reply:
done